internal/listener/listener.go

```diff
@@ -46,6 +47,8 @@ func (l *Listener) Run() error {
 }
 
 func (l *Listener) ProcessEvent(w http.ResponseWriter, req *http.Request) {
+	w.Header().Set("Server", "icinga-notifications/"+internal.Version.Version)
```

Comment:
It would be nice if this was not just specifically set in this response handler but instead consistent for all responses.

Have a look at the [`ListenAndServe()`](https://pkg.go.dev/net/http#ListenAndServe) function as well as the [`Handler`](https://pkg.go.dev/net/http#Handler) and [`ServeMux`](https://pkg.go.dev/net/http#ServeMux) types from the `net/http` packages and think about how you can put together/wrap them to achieve this.